code/modules/mob/new_player/new_player.dm

```diff
@@ -116,7 +116,7 @@
 
 			observer.started_as_observer = 1
 			close_spawn_windows()
-			var/turf/T = pickSpawnLocation(/mob/observer)
+			var/turf/T = pickSpawnLocation("Observer")
```

Comment:
Same there.
Why same location picks by path somethimes and somethimes with strings?

Reply:
But will that work?

Reply:
its working, i spend a lot of time reworking code

Reply:
It's not work.
You can test it.
![1111](https://user-images.githubusercontent.com/2458675/49989012-a8646b80-ff9a-11e8-85ab-6a1542b6d188.png)

Reply:
And it is better to return to the use of types.
This is a more reliable way.

Reply:
it works with string.
it wont work with types.
i told you already there are reasons behind this i wont redo this

Reply:
All the examples above is from code.
You say "It's not work with paths" but is use paths in code

Reply:
i will have to look at that

Reply:
should be all now. I couldnt reproduce datum/job/ai, this tell me if the problem persist